src/figuresClasses.ts

```diff
@@ -1,11 +1,112 @@
-export interface Figure {}
+export interface Figure {
+  color: string;
```

Comment:
create separate type for color

Reply:
Why create separate types for color and shape?